### ISSUE TITLE
There is no "Open in Portal/View Properties/Refresh" action for container app

### ISSUE THREAD

**Issue by v-ruizh** [microsoft/vscode-azurecontainerapps#142]
**OS:** All
**Build Version:** 20220706.3

**Repro Steps:**
1. Right click a container app -> check its context menu.
2. Check whether there is "Open in Portal/View Properties/Refresh" action for container app.

**Expect:**
There is "Open in Portal/View Properties/Refresh" action for container app.
![image](https://user-images.githubusercontent.com/45614674/177519516-d35cf314-09cf-4239-8521-ce58071a37c5.png)

**Actual:**
There is no "Open in Portal/View Properties/Refresh" action for container app.
![image](https://user-images.githubusercontent.com/45614674/177519334-8fee78d8-6159-454b-a0d8-8c6cb88172b7.png)

**More Info:**
There is no "View Properties" action for "Ingress" node.

**Comment by nturinski** [comment]
We don't see refresh still, but are the other two context menu items not appearing for you? I see both "Open in Portal" and "View Properties"

**Comment by v-ruizh** [comment]
Yes, verified on container apps extension 20230412.5, both "Open in Portal" and "View Properties" items exist in context menu, but still don't see the "Refresh" item.

**Comment by MicroFish91** [comment]
I believe this has been fixed for a while now
